### PR TITLE
Clarify usage of ?include_deleted on the times endpoints.

### LINKED ISSUE #52: Add times/uuid?include_deleted=true parameter
In conversation with @ElijahCaine and @LyonesGamer I learned that 
GET `/times/:uuid?include_deleted=true` is allowable.

All I can find in the docs currently is
GET `/times/:uuid?include_revisions=true`

Please add a section describing the uuid - include_deleted combination.

### PR DESCRIPTION
**Pull request by MorganEPatch**
Fixes #52.

### CODE REVIEW

File: source/draft_api.rst [hunk 1]
```diff
@@ -615,6 +615,62 @@ GET /activities?include_deleted=true
...
+        "duration": 20,
+        "user": "example-user",
+        "project": "gwm",
+        "activities": ["doc", "research"],
+        "notes": "Worked on documentation toward settings configuration.",
+        "issue_uri": "https://github.com/osuosl/ganeti_webmgr/issues/40",
+        "date_worked": "2015-04-18",
+        "created_at": "2014-06-12",
+        "updated_at": "2015-04-18",
+        "uuid": "aa800862-e852-4a40-8882-9b4a79aa3015",
```
Comment (athai) [inline source/draft_api.rst]: The uuid should be unique from the deleted object. Other than that, it looks good.

**Comment by athai** [comment]
+1

**Comment by Kennric** [comment]
Looks ok, but I think would be more clear with a note pointing out the record with the deleted_at set is visible - otherwise you have to read pretty carefully through the json to catch it.

**Comment by MorganEPatch** [comment]
To be fair, this does come after a section that already also includes `GET /activities?include_deleted=true` and `GET /projects?include_deleted=true`, which in turn is after a section that describes the behavior of `include_deleted`. This is just adding times to it for completeness (so no assumptions are made regarding the lack of example). I can make a more explicit note if that'd be preferred.

**Comment by Kennric** [comment]
We should point out that one of the returned records, when using include_deleted, is in fact a deleted record, and that we know this from the deleted_at field. We don't have to do that on every example, but at least one.

File: source/draft_api.rst [hunk 2]
```diff
@@ -587,6 +587,8 @@ GET /projects?include_deleted=true
       }
     ]
 
+Note that this now includes the Timesync project, which had previously been deleted.
```
Comment (pop) [inline source/draft_api.rst]: You should make this an RST not directive.

```
.. note::
    Notice that this now includes the Timesync project, which had previously been deleted.
```

File: source/draft_api.rst [hunk 3]
```diff
@@ -615,6 +617,69 @@ GET /activities?include_deleted=true
       }
     ]
 
+Note that this now includes the Meetings activity, which had previously been deleted.
```
Comment (pop) [inline source/draft_api.rst]: Also make this a note directive.

```
.. note::
    Notice that this now includes the Meetings activity, which had previously been deleted.
```

File: source/draft_api.rst [hunk 4]
```diff
@@ -615,6 +617,69 @@ GET /activities?include_deleted=true
...
+      "issue_uri": "https://github.com/osuosl/timesync/issues/52",
+      "date_worked": "2015-08-18",
+      "created_at": "2015-08-18",
+      "updated_at": null,
+      "deleted_at": "2015-10-12",
+      "uuid": "e283a2cd-39c6-4133-95ec-5bc10dd9a9ef",
+      "revision": 2
+    }
+
+As above, this time is deleted (note the deleted_at field), but instead of a 404, it
```
Comment (pop) [inline source/draft_api.rst]: Turn this into a note so people actually notice it.

File: source/draft_api.rst [hunk 5]
```diff
@@ -615,6 +617,69 @@ GET /activities?include_deleted=true
...
+        "date_worked": "2015-08-18",
+        "created_at": "2015-08-18",
+        "updated_at": null,
+        "deleted_at": "2015-10-12",
+        "uuid": "e283a2cd-39c6-4133-95ec-5bc10dd9a9ef",
+        "revision": 2
+      }
+    ]
+
+Note that this now includes the second time, which had previously been deleted.
```
Comment (pop) [inline source/draft_api.rst]: Turn this into a note so people actually notice it.

**Comment by pop** [comment]
+1

**Comment by Kennric** [comment]
Ok, that works. +1

**Pull request merged**